From: Len Brown <len.brown@intel.com>
Date: Thu, 16 Feb 2017 23:48:50 -0500
Subject: x86: intel-family.h: Add GEMINI_LAKE SOC
Git-commit: 311f77708f362c1af0a999ba5ebbbc062fdc5601
Patch-mainline: v4.11-rc1
References: bsc#1068032 CVE-2017-5754

Cc: x86@kernel.org
Signed-off-by: Len Brown <len.brown@intel.com>
[ Pulled in by a5b296636453 ("x86/cpufeature: Blacklist SPEC_CTRL/PRED_CMD on early Spectre v2 microcodes")]
Acked-by: Borislav Petkov <bp@suse.de>
---
 arch/x86/include/asm/intel-family.h |    1 +
 1 file changed, 1 insertion(+)

--- a/arch/x86/include/asm/intel-family.h
+++ b/arch/x86/include/asm/intel-family.h
@@ -60,6 +60,7 @@
 #define INTEL_FAM6_ATOM_MOOREFIELD	0x5A /* Annidale */
 #define INTEL_FAM6_ATOM_GOLDMONT	0x5C
 #define INTEL_FAM6_ATOM_DENVERTON	0x5F /* Goldmont Microserver */
+#define INTEL_FAM6_ATOM_GEMINI_LAKE	0x7A
 
 /* Xeon Phi */
 
